docs/sidebar.json

```diff
@@ -6,12 +6,14 @@
         "Grid Generation": "/tutorial.grids.html",
         "Geometry Validation": "/tutorial.geometry_validation.html",
         "Vector Zonal Stats": "/tutorial.vector_zonal_stats.html",
-        "Raster Zonal Stats": "/tutorial.raster_zonal_stats.html"
+        "Raster Zonal Stats": "/tutorial.raster_zonal_stats.html",
+        "DHS Processing": "/tutorial.dhs.html"
       }
     },
     "Grids": "grids.html",
     "Geometry Validation": "validation.html",
     "Raster Zonal Stats": "raster_zonal_stats.html",
-    "Vector Zonal Stats": "vector_zonal_stats.html"
+    "Vector Zonal Stats": "vector_zonal_stats.html",
+    "DHS processing": "dhs.html"
```

Comment:
hi @jtmiclat the generated html file is not `dhs.html` but based on the notebook name (snaked case w/o the number)  so the name of the file is `dhs_data.html` so the sidebar url needs to corrected.

Reply:
@butchtm Thanks for catching. I renamed the notebook to `dhs.ipynb` to be more consistent about it.